shared_model/interfaces/iroha_internal/transaction_sequence.cpp

```diff
@@ -22,13 +22,14 @@ namespace shared_model {
                          std::vector<std::shared_ptr<Transaction>>,
                          interface::types::HashType::Hasher>
           extracted_batches;
-      std::vector<TransactionBatch> batches;
+      types::BatchesCollectionType batches;
```

Comment:
I think, that line should be moved to 28, so it would be directly above its actual usage place